components/info/InfographicPage.tsx

```diff
@@ -45,13 +126,113 @@ export const InfographicPage: React.FC<InfographicPageProps> = ({
   targetsData,
   quizData,
   gameData,
-  ScrollComponent
+  ScrollComponent,
 }) => {
   return (
     <div className="infographic-page relative">
       <div id="header-section">
         <HeaderSection {...headerData} />
       </div>
+      <div className="p-4">
+        <h1 className="text-xl font-bold mb-4">SDG 6 Story App</h1>
```

Comment:
The a games look awesome, thanks Kevin!

Small thing, as we want "@/components/info/InfographicPage.tsx" to be clean from static data we should migrate it instead to the example infographic in "@/app/info/page.tsx".

Thanks again!